setup.py

```diff
@@ -11,8 +11,8 @@
 
 
 setup(
-    name="bwapi",
-    version="4.1.0",
+    name="bcr",
+    version="1.1.0",
```

Comment:
The version here is 1.1.0, but elsewhere it is 1.0.0 - is this intentional?

Reply:
Oops, no - will correct